ginga/misc/plugins/LineProfile.py

```diff
@@ -148,7 +148,10 @@ def new_image_cb(self, fitsimage, image):
         elif image is None:
             return False
         else:
-            self.build_axes()
+            try:
+                self.build_axes()
+            except:
+                pass
```

Comment:
This sort of construct, simply squelching the exception, may hide other exceptions that would be good to see.  Instead, we need to fix `build_axes()` so that it doesn't throw an exception in the 2D case.